contributing-guides/style-guide.md

```diff
@@ -64,6 +64,8 @@ Keep the following guidelines in mind when choosing tokens:
    But in a command like `wc -l {{file}}`, using `{{file}}` (without extension) is sufficient.
 7. If the example is clearer with an actual value rather than a generic placeholder, use the actual value.
    For example, use `iostat {{2}}` rather than `iostat {{interval_in_secs}}`.
+8. If a command performs irreversible changes to a file system or to user's devices, then write every example in a way that they cannot be unmindfully copy-pasted by the user.
+   For example, instead of `ddrescue --force --no-scrape /dev/sda /dev/sdb` write `ddrescue --force --no-scrape {{/dev/sdX}} {{/dev/sdY}}` and use the `{{/dev/sdXY}}` placeholder for *block devices* instead of `/dev/sda`.
```

Comment:
Did you mean `/dev/sda1` at the end of the sentence?

Reply:
Yep, sounds like a partition.